### PR TITLE
Added Meshery Playground CTA

### LINKED ISSUE #4129: [UX] Call to Action: Graphic design needed for Meshery Playground
### Current Behavior
As a hosted instance of Meshery running in the CNCF labs, the Meshery Playground is an easy way to use Meshery - https://play.meshery.io. The layer5.io website does not yet highlight this resource for visitors.

### Desired Behavior
The layer5.io website needs a fresh Call to Action (CTA)

### Screenshots / Mockups
Example CTA:
Screenshot 2023-04-24 at 4 19 19 PM

### Implementation
1. Design one or more new Call to Action graphics in Layer5 Community Figma page here -->  https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?node-id=0-1

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Savio629**
**Description**
Added Meshery Playground CTA

This PR fixes #4129

**Notes for Reviewers**
To check the cta, search on the following path (/resources/docker/configuring-highly-available-docker-swarm)
Once the location is confirmed, I'll move the cta to that specific page...

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 5742984be8f6e519d99f060147b2a14e9840ce9c at: https://64d8c6c77408bc112b89c94d--layer5.netlify.app

**Comment by Savio629** [comment]
// @thisiskaransgit @Ghat0tkach

**Comment by l5io** [comment]
🚀 Preview for commit 40e67928a8a72a56db78c1128de69a47a4ae673c at: https://64d8ceb49f1f93108bd7475d--layer5.netlify.app

**Comment by Ghat0tkach** [comment]
The CTA looks so good 🤩🤩🤩

**Comment by vishalvivekm** [comment]
@Savio629 Let's discuss it on the websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima), if you would :)

File: src/sections/Playground/playground-CTA.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        <div className="playground-callout">
+          <div className="dots-icon"/>
+          <div className="card-left">
+            <img className="meshery-title" src={MesheryTitle} alt="Meshery Title" loading="lazy" />
+            <div className="playground">Playground</div>
+          </div>
+          <div className="line"></div>
+          <div className="card-right">
+            <div>
+              <h4>Mesh Your Way to Success<br></br>Discover, validate, and visualize<br></br> Application with ease. Don't wait </h4>
```
Comment (leecalcote) [inline src/sections/Playground/playground-CTA.js:147]:
```suggestion
              <h4>Connect to live clusters<br />Discover, validate, and visualize<br />Kubernetes infrastructure with ease.</h4>
```

**Review by leecalcote — APPROVED** [review]
Let's get this used in a few other places on the site.

**Pull request merged**